content/docs/operating/storage.md

```diff
@@ -243,6 +275,16 @@ state, i.e. their current incomplete head chunk is not the one that is
 contained in the most recent checkpoint. This limit is configurable
 via the `storage.local.checkpoint-dirty-series-limit` flag.
 
+Especially on servers with many active series, the number of chunks waiting for
+persistency is inevitably high (as described in previous sections). The
+checkpoint size, and thus the time needed for checkpointing, essentially grows
+linearly with the number of active series. There is a clear trade-off between
```

Comment:
Yeah, that's the step in between. More series lead to more chunks waiting to persist lead to larger checkpoints. I'll try to word that more clearly.